cmd/thanos/query.go

```diff
@@ -42,11 +43,29 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application, name string
 
 	stores := cmd.Arg("store", "store APIs to get data from").Required().URL()
 
+	peers := cmd.Flag("cluster.peers", "Initial peers to join the cluster").Strings()
```

Comment:
lower case sentence start to be consistent with other flag descriptions.  (and in sidecar, store as well)